worker/handler/handler.go

```diff
@@ -277,6 +360,20 @@ func (h *Handler) RunStart() (ch *sb.Channel, err error) {
 
 	// create sandbox if needed
 	if h.sandbox == nil {
+		// if estimated mem usage exceeds the hard limit, the handler would queue itself
+        hms.sbCreatingLock.Lock()
```

Comment:
Do we need this since we have a central scheduler, which we communicate with via the chan?

Reply:
Removed in the new commit.